.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
   conda config --add channels conda-forge &&
   conda update --yes conda &&
   conda update --all -y python=$TRAVIS_PYTHON_VERSION &&
-  conda install -y nipype &&
+  conda install -y nipype graphviz &&
```

Comment:
I think that should go here: https://github.com/ashgillman/nipype/blob/4c3a232b5d70da3638ff54dea173bcf3adf55acb/.travis.yml#L22 as an apt package.